WalletWasabi/Crypto/ZeroKnowledge/LinearRelation/Statement.cs

```diff
@@ -1,12 +1,13 @@
 using NBitcoin.Secp256k1;
-using System;
 using System.Collections.Generic;
 using System.Linq;
+using WalletWasabi.Crypto.Groups;
 using WalletWasabi.Helpers;
+using WalletWasabi.Crypto.ZeroKnowledge;
 
 namespace WalletWasabi.Crypto.ZeroKnowledge.LinearRelation
 {
-	public class Statement
+	public class Statement : IStatementDescription
```

Comment:
Should `Statement` be `StatementDescription` then?

Reply:
nope, this is a narrower interface, only what is necessary for the transcript to get the metadata it needs to hash.

this is a compatibility shim, since there already exists a `Statement` class, in #4303 that would eventually be deleted so this interface can be removed after then and `Transcript` specialized to `LinearRelation.Statement` like the Fiat-Shamir `Knowledge` and `Statement` classes.

Reply:
FWIW the rationale for that was to first add the code, then have all of the existing tests point at the new version, i think that would provide better continuity when reading the git logs of the tests, since if it's done in the other order the failing tests would need to be deleted and reintroduced later, or everything done in one giant commit.